libethereum/BlockChainSync.cpp

```diff
@@ -52,6 +30,10 @@ std::ostream& dev::eth::operator<<(std::ostream& _out, SyncStatus const& _sync)
 namespace  // Helper functions.
 {
 
+unsigned constexpr c_maxPeerUknownNewBlocks = 1024; /// Max number of unknown new blocks peer can give us
```

Comment:
Pedantic mode: should be `constexpr unsigned`.

Reply:
Why do we put `constexpr `before the type but `const `after the type?